RevenueCatUI/Views/RemoteImage.swift

```diff
@@ -19,42 +19,80 @@ struct RemoteImage: View {
     let url: URL
     let aspectRatio: CGFloat?
     let maxWidth: CGFloat?
+    let lowResSuffix: String?
```

Comment:
I've been wondering... Would it make more sense to have the backend provide the name of the low res image on a different field if available, instead of relying in a convention like this? So basically RemoteImage would have 2 URLs, one for the high res and one for the low res version, and we need to pass both (if available) from the backend data. Wdyt?

Reply:
Ah yes, I think it would!

Reply:
Implemented here, can you take a look? https://github.com/RevenueCat/purchases-ios/pull/3906/commits/16c7473a42a94195be279ab85a3c13e2df9f7b49